spec/javascript/RestaurantInfoTile.js

```diff
@@ -0,0 +1,35 @@
+import RestaurantInfoTile from "../../app/javascript/tiles/RestaurantInfoTile"
+import CategoryTile from "../../app/javascript/tiles/CategoryTile"
+
+describe('RestaurantInfoTile', () => {
+  let text,
+      wrapper;
+
+  beforeEach(() => {
+    wrapper = mount(
+      <RestaurantInfoTile
+        restaurant={{ name: "Papa Johns", street: "123 Food Lane", city:"Boston", state:"MA", zip:"02110", categories:[{name:"Fast Food"}] }}
```

Comment:
This is okay, but in the future, try to avoid long lines by using line breaks between adjacent key/value pairs and perhaps saving this whole object in a variable above the `wrapper = mount(` line